documentation/asciidoc/computers/config_txt/memory.adoc

```diff
@@ -24,9 +24,11 @@ The recommended maximum values are as follows:
 | `384`
 
 | 1GB or greater
-| `512`, `256` on the Raspberry Pi 4
+| `512`, `76` on the Raspberry Pi 4
 |===
 
+IMPORTANT: The default camera stack (libcamera2) on Raspberry Pi OS - Bullseye uses Linux CMA memory to allocate buffers instead of GPU memory so there is no benefit in increasing the GPU memory size.
```

Comment:
I'm not sure if we need the "Bullseye" signifier here, as the assumption is generally that our documentation is just about the latest version of Raspberry Pi OS, which is currently Bullseye? :shrug:
(then again, there's also #2314 )  ping @aallan

Reply:
Think it's probably fine in context.